manifests/init.pp

```diff
@@ -57,7 +57,7 @@
 
   file { '/root/.puppetlabs/bolt/bolt-project.yaml':
     ensure  => file,
-    content => to_yaml( {
+    content => to_yaml ({
         'modulepath' => [
           '/etc/puppetlabs/code/environments/production/site-modules',
```

Comment:
IM a little dubious about the hardcoding to production environment, i know we talked about that in enablement, i think that's fine, however have we tested if there are issues if a "site-modules" folder does not exist? as that content folder is not product its just an adopted style